cpu/native/startup.c

```diff
@@ -495,7 +495,7 @@ __attribute__((constructor)) static void startup(int argc, char **argv, char **e
 #endif
 #ifdef MODULE_SOCKET_ZEP
             case 'z':
-                _zep_params_setup(optarg, zeps++);
+                _zep_params_setup(strdup(optarg), zeps++);
```

Comment:
Still would prefer if the `strdup` was moved into `_zep_params_setup()`.

Reply:
(with a comment)

Reply:
Ok, moved it there and amended the commit message too.